tests/knownKeys

```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
 config_screenBrightnessForVrSettingDefault
 config_screenBrightnessForVrSettingMinimum
 config_screenBrightnessForVrSettingMaximum
+flingboost_param_value
+scrollboost_param_value
+launchboost_param_value
+actlaunchboost_param_value
+launchboost_packing_param_value
+panelview_flingboost_param_value
+animationboost_param_value
+config_enableCpuBoostForOverScrollerFling
+config_enableCpuBoostForScroller
+config_enableCpuBoostForAppLaunch
+config_enableCpuBoostForActLaunch
+config_disablePacking
+config_enableCpuBoostForPanelViewFling
+config_enablePerfBoostForAnimation
+carrier_vt_available_bool
+config_enableGeofenceOverlay
+launchboost_timeout_param
+actlaunchboost_timeout_param
+disablepacking_timeout_param
+animationboost_timeout_param
+config_wifi_supplicant_scan_interval
```

Comment:
This doesn't sound like an hardware feature to me, just ROM's decision.
And indeed, Pixels/Nexus doesn't set this.